ThirdParty.json

```diff
@@ -2,96 +2,119 @@
   {
     "name": "Async++",
     "url": "https://github.com/Amanieu/asyncplusplus",
-    "license": "MIT"
+    "version": "756893feb9e69c098225d5a454a668a2c139e4be",
```

Comment:
@kring To confirm - Are we using commits throughout because we don't use release versions of the libraries?

Reply:
Yeah, I used commits when there is no specific version number we're using. In some cases that's because the library doesn't even release versions.